services/property-svc/internal/property/models/property.go

```diff
@@ -15,7 +15,7 @@ type Property struct {
 	IsArchived  bool
 
 	SetID         uuid.NullUUID
-	FieldTypeData FieldTypeData
+	FieldTypeData *FieldTypeData
```

Comment:
Properties have FieldTypeData. Always. The FieldTypeData itself might be full of only nil values though. Here we have a similar issue as we had with SelectOption. I think we should have a seperate UpdateProperty model, for which FieldTypeData may be nil, when it should not be changed

Reply:
I don't see the need for an own UpdatePropertyModel we are not facing the similar issue as with SelectOption if I understand correctly.

Reply:
I get why it is semantically wrong, that's why i fixed that in my latest commit.
> we are not facing the similar issues

Reply:
I get why it is semantically wrong, that's why i fixed that in my latest commit.
> we are not facing the similar issues 

With that I just wanted to express that we don't need an extra `UpdateProperty` model atm where `FieldTypeData` is optional, all update operations on SelectData (upsert, remove, allowFreetext) are taken care of.

Reply:
but how do we model no change to FieldTypeData?

Reply:
`FieldTypeData` is of type oneof an thus dont has to be set in the request.
If`FieldTypeData` is i.e `SelectData` in the request the operations (upsert, remove, allowFreeText) will trigger the according Events and changes will be applied to the aggregate. 

https://github.com/helpwave/services/blob/daefb51f3e63a06d9973ccf381d33a81cc8a510c/services/property-svc/internal/property/api/grpc.go#L113

If FieldTypeData is not set, no events will be triggerted